### ISSUE TITLE
[FR] Integration limits

### ISSUE THREAD

**Issue by KronosTheLate** [SciML/SymbolicNumericIntegration.jl#23]
It would be sweet if it was possible to supply integration limits to automate the step of substituting them in and evaluating manually ^_^

**Comment by Dale-Black** [comment]
I agree with the above comments. Right now I have a simple extension for my code

```julia
function SymbolicNumericIntegration.integrate(f, x, x0, x1)
	F = integrate(f, x)[1]
	F = substitute(F, x => x1) - substitute(F, x => x0)
end
```

Would a PR be of interest?

**Comment by B-LIE** [comment]
It would be interesting, but does this solution handle the possibility that either of `x0` or `x1` are symbolic variables? When I played around with it, `substitute` didn't work with symbolic variables.

**Comment by shahriariravanian** [comment]
Implemented in the new version (1.2).
